pkg/client/client_test.go

```diff
@@ -84,6 +84,7 @@ func newTestClient(p4RuntimeClient *fakeP4RuntimeClient, p4Info *p4_config_v1.P4
 		P4RuntimeClient: p4RuntimeClient,
 		deviceID:        1,
 		electionID:      p4_v1.Uint128{High: 0, Low: 1},
+		role:            &p4_v1.Role{Config: nil},
```

Comment:
again, why not `nil`?

Reply:
Ah, I see what you meant. Sorry I misread it.

Reply:
Let me fix this in the next patch.